### PR TITLE
Move to Bootstrap3 with GitHub theme

### PR DESCRIPTION
**Pull request by takezoe**

### CODE REVIEW

**Comment by McFoggy** [comment]
@takezoe will you expose/document the changes that are required by plugins (if any).

**Comment by takezoe** [comment]
@McFoggy Migration guides for Bootstrap2 to 3 might be helpful.

File: src/main/twirl/gitbucket/core/helper/attached.scala.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         var attachFile = (file.type.match(/image\/.*/) ? '\n![' + file.name.split('.')[0] : '\n[' + file.name) +
           '](@baseUrl/@owner/@repository/_attached/' + id + ')';
         $('#@textareaId').val($('#@textareaId').val() + attachFile);
-        $(file.previewElement).prevAll('div.dz-preview').addBack().remove();
+        $(file.previewElement).prevAll('div.dz-preview').addBack().remove();2
```
Comment (nazoking) [inline src/main/twirl/gitbucket/core/helper/attached.scala.html]: type miss?

Comment (takezoe) [inline src/main/twirl/gitbucket/core/helper/attached.scala.html]: @nazoking Fixed. Thanks a lot.

**Pull request merged**